src/loguru_parallel/listen.py

```diff
@@ -51,15 +52,30 @@ def stop(self) -> None:
         if self._thread is None:
             return
         super().stop()
-
-    def _monitor(self) -> None:
-        """Monitor the queue for records, and ask the handler to deal with them."""
-        super()._monitor()
         self._sink_logger.complete()
         print("Loguru Listener stopped.")
 
 
 def loguru_enqueue_and_listen(handlers: list[dict[str, Any]]) -> LoguruQueueListener:
```

Comment:
Just a thought, but could we use `with` context to limit the impact of changing the logger instance and ensure that resources are freed when the context is exited? It's nice that it's easy to just call a function, but we really need to be careful how we use `loguru_enqueue_and_listen` since it overwrite the `loguru.logger` configuration up to that point, right?

Reply:
Great idea. Yes, I think it should be possible to implement this as a context-manager, but the implementation won't be trivial.
Let me try doing this in another PR!